src/configuration-store/memory.store.ts

```diff
@@ -0,0 +1,60 @@
+import { IAsyncStore, IConfigurationStore, ISyncStore } from './configuration-store';
+
+export class MemoryStore<T> implements ISyncStore<T> {
+  private store: Record<string, string> = {};
```

Comment:
Any reason to serialize to strings?

Reply:
Agreed this could/should be
```suggestion
  private store: Record<string, T> = {};
```
right?